lib/process/local-repl-process.coffee

```diff
@@ -132,7 +131,6 @@ class LocalReplProcess
 
   interrupt: ->
     @conn.interrupt()
-    @appendText("Interrupting")
```

Comment:
I liked having feedback here to know the interrupt call was sent. Any particular reason you removed this?